src/plugins/VMProfileLinuxSupportPlugin/VMProfileLinuxSupportPlugin.c

```diff
@@ -171,7 +177,7 @@ primitiveDLSymInLibrary(void)
 	symName = malloc(sz + 1);
 	strncpy(symName, firstIndexableField(nameObj), sz);
 	symName[sz] = 0;
-	lib = dlopen(libName, RTLD_LAZY | RTLD_NODELETE);
```

Comment:
Naive question... what is wrong with RTLD_NODELETE?  It seems like it could be useful to prevent memory leaks, unless we're *sure* all threads are destroyed before calling dlclose() 
https://mta.openssl.org/pipermail/openssl-commits/2016-June/007938.html